types/postcss-prefix-selector/postcss-prefix-selector-tests.ts

```diff
@@ -1,4 +1,4 @@
 import postcssPrefixSelector = require('postcss-prefix-selector');
 
-// $ExpectType (root: any) => string | undefined
+// $ExpectType (root: Root_) => void
```

Comment:
Is this `_` intended? I've never seen that, just curious

Reply:
Ah, that's the type `postcss`'s class `Root` get resolved to. https://github.com/postcss/postcss/blob/b0efcc0bc5aee44651a97cd648f66448d9bb3d3f/lib/root.d.ts#L84

Keeping it `Root` won't pass the test. Other plugins also have this: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/05094b6df4fb6f4404333725f13d63dd8aabeb34/types/rtlcss/rtlcss-tests.ts#L9 https://github.com/DefinitelyTyped/DefinitelyTyped/blob/05094b6df4fb6f4404333725f13d63dd8aabeb34/types/postcss-js/postcss-js-tests.ts#L15